stable/traefik/README.md

```diff
@@ -179,3 +181,7 @@ dashboard:
     basic:
       test: $apr1$H6uskkkW$IgXLP6ewTrSuBkTrqE8wj/
 ```
+
+### Proxy Protocol
+
+In situations where Traefik lives behind an Internet-facing loadbalancer (like and AWS ELB) and you still want it to see the actual source IP of the visitor instead of the internal port of the loadbalancer, you can enable the loadbalancer to use the Proxy protocol to talk to Traefik. This effectively makes the loadbalancer transparant, as Traefik will still get the actual visitor IP address for each request. This only works if Traefik knows it's receiving traffic via the Proxy Protocol and the loadbalancer IP addresses need to be whitelisted as well.
```

Comment:
> like and AWS ELB

s/and/an/

Reply:
> instead of the internal port of the loadbalancer

s/port/IP/

Reply:
> This only works if Traefik knows it's receiving traffic via the Proxy Protocol and the loadbalancer IP addresses need to be whitelisted as well.

If either party (proxy or Traefik) is using proxy protocol and the other is not, things will break badly. (Bad gateway maybe? I cannot recall how it manifests.) It might be worth it to caution users about this.